Tests/BacktraceCrashLoopDetectorTests.swift

```diff
@@ -0,0 +1,52 @@
+import XCTest
+
+import Nimble
+import Quick
+@testable import Backtrace
+
+final class BacktraceCrashLoopDetectorTests: QuickSpec {
+
+    override func spec() {
+        describe("Crash Loop Detector") {
+
+            let crashLoopDetector = BacktraceCrashLoopDetector()
+            let eventsCount = BacktraceCrashLoopDetector.consecutiveCrashesThreshold
+            let timeIntervalStep = 200
+            var isCrashLoop = false
```

Comment:
Why define it here? I would just keep it local to the tests. There's no reason to persist across test.